bundle/jwtbundle/bundle.c

```diff
@@ -1,3 +1,28 @@
+
+/**
+
+(C) Copyright 2020-2021 Hewlett Packard Enterprise Development LP
+
+ 
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may
+not use this file except in compliance with the License. You may obtain
+a copy of the License at
+
+ 
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+
+**/
+
```

Comment:
This review is for everyone!
What do you think about using the **doxygen format**?
example:
```
/**
 *
 *   (C) Copyright 2020-2021 Hewlett Packard Enterprise Development LP
 *
 *  Licensed under the Apache License, Version 2.0 (the "License"); you may
 *  not use this file except in compliance with the License. You may obtain
 *  a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 *  Unless required by applicable law or agreed to in writing, software
 *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
 *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 *  License for the specific language governing permissions and limitations
 *  under the License.
 *
 */

Reply:
I second this

Reply:
Done